README.md

```diff
@@ -44,62 +141,57 @@ On Ubuntu, an install script is available on each release [here](https://github.
 ./semgrep-v0.9.0-ubuntu-generic.sh
 ```
 
-To try `semgrep` without installation, you can also run it via [Docker](https://docs.docker.com/install/):
+To try Semgrep without installation, you can also run it via [Docker](https://docs.docker.com/install/):
 
 ```
 docker run --rm -v "${PWD}:/home/repo" returntocorp/semgrep --help
 ```
 
-## Usage
+## Terminology
```

Comment:
Meta comment. By this point in the README it's feeling very dense. I'd love to do a design pass to see if we can make it feel lighter or cut more content.